packages/helper/src/slack/index.ts

```diff
@@ -87,7 +87,9 @@ async function sendSlackFeedbackMessage(slackEmail: ISlackFeedbackMsg): Promise<
 }
 
 async function sendDonationNotificationToSlack(donationInfo: IDonationSlack): Promise<boolean> {
-	const { service, userDonation, donor, wishCard, amount } = donationInfo;
+	const { service, donor, wishCard, amount } = donationInfo;
+	let { userDonation } = donationInfo;
+	userDonation = userDonation || 0.0;
```

Comment:
you don't need the re assignment here, you can do the check in the template string
```/wishcards/admin/${wishCard._id}, amount: ${amount} with ${userDonation || 0.0} for us`,```